phpstan.neon.dist

```diff
@@ -45,6 +45,8 @@ parameters:
 		- '#Return type \(bool\) of method CodeIgniter\\Log\\Logger::(emergency|alert|critical|error|warning|notice|info|debug|log)\(\) should be compatible with return type \(null\) of method Psr\\Log\\LoggerInterface::(emergency|alert|critical|error|warning|notice|info|debug|log)\(\)#'
 		- '#Return type \(bool\) of method CodeIgniter\\HTTP\\Files\\UploadedFile::move\(\) should be compatible with return type \(CodeIgniter\\Files\\File\) of method CodeIgniter\\Files\\File::move\(\)#'
 		- '#Return type \(bool\) of method CodeIgniter\\Test\\TestLogger::log\(\) should be compatible with return type \(null\) of method Psr\\Log\\LoggerInterface::log\(\)#'
+		- '#Parameter \#2 \$assoc of function json_decode expects bool, null given#'
+	reportUnmatchedIgnoredErrors: false
```

Comment:
I think this should be set as true so that we can remove errors no longer applicable.

Reply:
yes, it currently shows notice in php < 8